docs/docs/shards/index.md

```diff
@@ -164,4 +164,596 @@ Showing how we mix the Clojure-like language as a template language
+
+=== "Code"
+
+    ```clojure linenums="1"
+    [2.0 10.0]
+    (Math.Mean :Kind Mean.Arithmetic)
+    (Log)   ;; AM => 6
+
+    [2.0 10.0]
+    (Math.Mean :Kind Mean.Geometric)
+    (Log)   ;; GM => 4.47214
+
+    [2.0 10.0]
+    (Math.Mean :Kind Mean.Harmonic)
+    (Log)   ;; HM => 3.33333
+    
+    [2.0 10.0]
+    (Math.Mean 
+    ;; AM is default, anything else will throw error
+        ;; :Kind 123
+        )   
+    (Log)   ;; AM => 6
+    ```
+
+=== "Output"
+
+    ```
+    [info] [2022-07-22 15:35:00.868] [T-15316] [logging.cpp::55] [mywire] 6
+    [info] [2022-07-22 15:35:00.881] [T-15316] [logging.cpp::55] [mywire] 4.47214
+    [info] [2022-07-22 15:35:00.882] [T-15316] [logging.cpp::55] [mywire] 3.33333
+    [info] [2022-07-22 15:35:00.883] [T-15316] [logging.cpp::55] [mywire] 6
+    ```
+
+### Bool
+
+Type `Bool` allows only two values - `true` or `false`.
+In that sense it can be thought of as a special case of an [`Enum`](#-enum) data type.
+
+Consider the shard [`(Is)`](https://docs.fragcolor.xyz/shards/General/Is/). This shard compares its input and the value in the `:Value` parameter for equality. After the comparison it needs to communicate its result in a yes/no format (yes values are equal; no values are not equal).
+
+```clojure linenums="1"
+(Is
+  :Value [(Any)]
+)
+```
+
+To allow the shard to do this its output type is defined as a `Bool`. If the values are equal this shard emits `true` as its output, if the values are inequal it emits `false`. No other output is allowed.
+
+=== "Code"
+
+    ```clojure linenums="1"
+    100 (Is :Value (* 10 10))
+    (Log)   ;; Is equal => true
+
+    [20] (Is :Value 20)
+    (Log)   ;; Is not equal => false
+    ```
+
+=== "Output"
+
+    ```
+    [info] [2022-07-22 18:38:24.383] [T-25360] [logging.cpp::55] [mywire] true
+    [info] [2022-07-22 18:38:24.395] [T-25360] [logging.cpp::55] [mywire] false
+    ```
+
+### Int
+
+Type `Int` (or its alias `int`) defines a 64-bit signed integer.
+
+64 bits of memory allows this data type to store integer values ranging from -9,223,372,036,854,775,808 to +9,223,372,036,854,775,807 (no decimals).
+
+An `Int` value looks like this: `(Int 2)`.
+It may also be represented without the keyword `Int`, with just the integer value: `2`.
+
+[`(Math.Add)`](https://docs.fragcolor.xyz/shards/Math/Add/) is an example of a shard that uses this data type for its input, output, and `:Operand` parameter.
+
+=== "Code"
+
+    ```clojure linenums="1"
+    (Int 3) (Math.Add (Int 2))
+    (Log)   ;; Int output => 5
+
+    3 (Math.Add 2)
+    (Log)   ;; Int output => 5
+    ```
+
+=== "Output"
+
+    ```
+    [info] [2022-07-22 21:20:18.771] [T-4568] [logging.cpp::55] [mywire] 5
+    [info] [2022-07-22 21:20:18.782] [T-4568] [logging.cpp::55] [mywire] 5
+    ```
+
+### Int2
+
+Type `Int2` (or its alias `int2`) defines a vector of two [`Int`](#-int) type numbers.
```

Comment:
```suggestion
Type `int2` (or its alias `Int2`) defines a vector of two [`int`](#-int) type numbers.
```

Etc.

Reply:
Fixed in commit #[190ce35f2cccc0d0714284d4e929d35f5ee6c3f1](https://github.com/fragcolor-xyz/shards/pull/325/commits/190ce35f2cccc0d0714284d4e929d35f5ee6c3f1).